privaterelay/settings.py

```diff
@@ -1171,6 +1177,7 @@ def set_index_cache_control_headers(
 if USE_SILK:
     SILKY_PYTHON_PROFILER = True
     SILKY_PYTHON_PROFILER_BINARY = True
+    SILKY_PYTHON_PROFILER_RESULT_PATH = ".silk-profiler"
```

Comment:
(question) Can you elaborate on why this config value was added? Also, if there is an appropriate documentation to go with these configs that would be nice.

(suggestion, non-blocking) Looking through the code to enable `silk` on my local I have to have `DEBUG` and `HAS_SILK` be true. To get `HAS_SILK` I need to install `silk`. Perhaps we should consider creating a `requirements-dev.txt` file so that we are not manually reading code to see what packages we need to install?

Reply:
This change broke CI and I'd rather not debug it in this PR.